.github/workflows/dagster.yml

```
@@ -57,10 +57,9 @@ on:
         required: true
 
 jobs:
-
   run-docker-workflow:
     # only run on synchronize PR event, meaning something has been pushed to the PR branch
-    if: github.event.action == 'synchronize'
+    if: ${{ github.event.action == 'synchronize' || github.event.action == 'push' }}
```

Should we relax the event actions to match the dev-env deploy? I think the other missing events right now are when a PR is opened (`opened` event) or reopened (`reopened` event) with the `dev-env` tag already added and when a `dev-env` label was removed - a bunch of commits added, and then the `dev-env` tag is added (`labeled` event)
```suggestion
    if: ${{ (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'dev-env')) ||  github.event.action == 'push' }}
```